bql/semantic/expression.go

```diff
@@ -249,6 +250,44 @@ func (e *comparisonForNodeLiteral) Evaluate(r table.Row) (bool, error) {
 	}
 }
 
+// comparisonForTimeLiteral is the internal representation of an expression of comparison between a binding and a time literal.
+type comparisonForTimeLiteral struct {
+	op OP // operation.
```

Comment:
For naming a general rule of thumb I follow and recommend is "a name should be proportional to the size of its scope and inversely proportional to the number of times that it is used within that scope"

Struct fields are usually passed around, so it's usually ok to have explicit names to understand the code without having to rely on the struct comments.

I'd recommend using: operation, leftBinding and rightTimeLiteral.  Long names should follow the Go convention of MixedCaps(https://golang.org/doc/effective_go.html#mixed-caps)

Reply:
Thank you for the comments Jose. I do agree with you.
Here above I was just following the standard of the file. In this `expression.go` we have some similar structs following this convention, but I already discussed about it with Thiago and Kloss and we decided to refactor the entire file to improve readability, especially these struct attribute names. That is why I opened the PR https://github.com/google/badwolf/pull/139.
We decided to first merge the current opened PRs to master, including this 134 here, and then centralize all the refactoring in PR 139 so it will be easier to keep the same pattern for all the comparison functions.
Then, I will be doing this renaming for `rightTimeLiteral`, `leftBinding` etc there as I already started to do with the other similar structs we have there.

Reply:
LGTM